pkg/scheduler/scheduler.go

```diff
@@ -199,10 +199,12 @@ func (s *Scheduler) processAllocationReleaseByAllocationKey(allocationAsksToRele
 				log.Logger().Info("release allocation",
 					zap.String("allocation", toRelease.Allocationkey),
 					zap.String("appID", toRelease.ApplicationID),
-					zap.String("message", toRelease.Message))
+					zap.String("message", toRelease.Message),
+					zap.Int("ReservedToRelease", reservedAsks))
```

Comment:
`ReservedToRelease` should be `reservedAskReleased`

Reply:
Done